frappe/core/doctype/user/user.py

```diff
@@ -205,7 +205,7 @@ def send_password_notification(self, new_password):
 						_update_password(user=self.name, pwd=new_password,
 							logout_all_sessions=self.logout_all_sessions)
 
-					if not self.flags.no_welcome_mail and self.send_welcome_email==1:
+					if not self.flags.no_welcome_mail and cint(self.send_welcome_email):
```

Comment:
@Thunderbottom Thanks for the suggestion. I was not aware of the cint(), will definitely use it in my coding.